virttest/qemu_vm.py

```diff
@@ -104,14 +104,21 @@ def qemu_proc_term_handler(vm, monitor_exit_status, exit_status):
     Callback function to detect QEMU process non-zero exit status and
     push VMExitStatusError to background error bus.
 
-    :param vm: VM name.
+    :param vm: VM object.
     :param monitor_exit_status: True to push VMUnexpectedExitError instance
         with calltrace to global error event bus.
     :param exit_status: QEMU process exit status.
     """
+    images_offline_snapshots = vm.devices.images_offline_snapshots
+    if images_offline_snapshots and vm.clean_images_offline_snapshots:
+        for file in images_offline_snapshots.values():
+            logging.info("Removing the temporary file: %s." % file)
```

Comment:
Suggest either removing this log or using `logging.debug`.

Reply:
Thanks, have updated.